game/rooms/room03/esc/room03_bridge.esc

```diff
@@ -1,6 +1,13 @@
 # THIS ROOMS FEATURES AN OPEN BRIDGE WITH A ***BROKEN*** BUTTON TO USE IN ORDER TO OPEN
 
-:setup
+
```

Comment:
Hm. Isn't this part of the preparation for the scene? Doesn't that make the bridge flip up or down visibly to the player? (Applicable to all other changed esc files as well)

Reply:
Indeed this is not correct. Fixing.